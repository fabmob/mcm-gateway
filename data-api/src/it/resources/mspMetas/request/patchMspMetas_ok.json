{
  "name": "Kapten name",
  "operator": "Kapten operator"
}