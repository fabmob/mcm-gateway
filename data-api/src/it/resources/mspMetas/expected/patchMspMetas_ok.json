{
  "hasParking": false,
  "hasStationStatus": true,
  "primaryColor": "#0054BB",
  "type": "TROTTINETTE",
  "operator": "Kapten operator",
  "url": "https:\/\/www.cityscoot.eu\/paris\/",
  "logoUrl": "@MAAS_API_DIRECTORY_ICON@\/icon_cityscoot@3x.png",
  "priceList": {
    "duration": [
      {
        "unit": 5,
        "fixedFare": 6,
        "farePerUnit": 7,
        "priceListItemId": "87930fdf-34c1-41c9-885e-6ce66505b598",
        "lowerPriceLimit": 4,
        "upperPriceLimit": 9
      }
    ],
    "parkingForbiddenFee": 3,
    "distance": [
      {
        "unit": 5,
        "fixedFare": 6,
        "farePerUnit": 7,
        "priceListItemId": "86d370e4-06ce-4dd7-8c4c-2d7f80d46f3d",
        "lowerPriceLimit": 4,
        "upperPriceLimit": 9
      }
    ],
    "priceListId": "22808a15-1b7b-4605-9764-3c71a3f05ae4",
    "outOfBoundFee": 5,
    "comment": ""
  },
  "mspId": "b814c97e-df56-4651-ac50-11525537964f",
  "hasVehicle": null,
  "hasStation": false,
  "hasPrefParkingZone": false,
  "urlWebview": false,
  "isEnabled": null,
  "name": "Kapten name",
  "hasSpeedLimitZone": false,
  "logoFormat": "PNG",
  "hasNoParkingZone": false,
  "hasHold": false,
  "hasOperatingZone": false,
  "secondaryColor": "#0054BB"
}